src/darwin/Framework/CHIPTests/MTRXPCListenerSampleTests.m

```diff
@@ -26,6 +26,7 @@
 #import "MTRTestKeys.h"
 #import "MTRTestResetCommissioneeHelper.h"
 #import "MTRTestStorage.h"
+#import "MTRUnfairLock.h"
```

Comment:
Looks like this fails to compile, because you can't do C++ bits in Objective-C (not Objective-C++).

Should probably just revert the changes to this file and MTRErrorTests.m for now.